nextflow.config

```diff
@@ -63,6 +63,7 @@ params {
 
     // Alignment
     aligner                    = 'star_salmon'
+    use_sentieon               = false
```

Comment:
Since this only applies to STAR, should we call it `use_sentieon_star` instead?

Reply:
9450a5cf